packages/langium/src/test/parse-helper.ts

```diff
@@ -0,0 +1,33 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { LangiumDocumentConfiguration } from '../documents/document';
+import { BuildResult, DocumentBuilder } from '../documents/document-builder';
+import { LanguageMetaData } from '../grammar/language-meta-data';
+import { LangiumServices } from '../services';
+
+export interface ParseHelper {
+    parse(input: string): BuildResult
+}
+
+export class DefaultParseHelper implements ParseHelper {
```

Comment:
While we're transforming classes to functions in the context of tests, shall this be a function as well?